includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1556,11 +1557,34 @@ public function update_subscription_item( $wcpay_subscription_item_id, $data ) {
 	 * @throws API_Exception If payment method does not exist.
 	 */
 	public function get_payment_method( $payment_method_id ) {
-		return $this->request(
+		// A fallback for sites, which migrated from Stripe to WCPay.
```

Comment:
Let's expand a bit more on what's hapenning here, and why. E.g. "Sites migrated from Stripe to WC Payments may have old source objects in place of payment methods, so another API endpoints need to be used for fetching payment method details".

Reply:
Done In https://github.com/Automattic/woocommerce-payments/pull/4663/commits/0ad203a85bf1c300d838519ab3eeec2d9e39dbd1 👍